fofix/game/song/song.py

```diff
@@ -325,20 +325,31 @@ def getScores(self, part):
             scores = self._get("scores", str, "")
             scores_ext = self._get("scores_ext", str, "")
 
+        if scores:
+            try:
+                scores = cerealizer.loads(binascii.unhexlify(scores))
+            except:
+                log.error("High scores lost! Can't parse scores = %s" % scores)
+                scores = None
+
         if not scores:
             return
 
-        scores = cerealizer.loads(binascii.unhexlify(scores))
         if scores_ext:
-            scores_ext = cerealizer.loads(binascii.unhexlify(scores_ext))
+            try:
+                scores_ext = cerealizer.loads(binascii.unhexlify(scores_ext))
+            except:
```

Comment:
You should specify the exception (use `Exception` instead).

Reply:
As above.